src/test/java/uk/gov/hmcts/reform/idam/health/command/ReplicationCommandProbeTest.java

```diff
@@ -133,4 +133,13 @@ public void testProbe_HostLaggingAndOthersOkayButNoThreshold() throws Interrupte
         assertThat(result, is(true));
     }
 
+    @Test
+    public void testProbe_HostWithNaDelay() throws InterruptedException, ExecutionException, IOException {
+        List<String> commandOutput = Collections.singletonList("dc=reform,dc=hmcts,dc=net\ttest-host:4444\t27259\ttrue\t24501\t1265\t8989\t N/A\ttrue");
+        TextCommandRunner.Response testResponse = new TextCommandRunner.Response(commandOutput, null);
+        when(textCommandRunner.execute(eq("test-template test-user test-password".split(" ")), eq(20000L))).thenReturn(testResponse);
```

Comment:
not sure why splitting a string instead of using array but ok

Reply:
I just copied another test that was doing it.